ansible/roles/debops.gitlab_runner/defaults/main.yml

```diff
@@ -139,7 +139,7 @@ gitlab_runner__fqdn: '{{ ansible_fqdn }}'
 #
 # Fully Qualified Domain Name of the GitLab CI host which will control the
 # runner operation.
-gitlab_runner__api_fqdn: 'code.{{ gitlab_runner__domain }}'
+gitlab_runner__api_fqdn: '{{ gitlab__fqdn|d("code." + gitlab_runner__domain) }}'
```

Comment:
This is the same case as with `gitlab_domain` and won't work as you expect in the default configuration.